### DISCUSSION TITLE
Doc string Style: single space prefix?

### DISCUSSION THREAD

**Discussion by MichaelQuaMan** [pytransitions/transitions#598] [Q&A]
Noticed many of the project doc strings start with a space before the first word. 

I’ve never seen this style. Looks like a conscious choice.

I’d love to know why? 

Seems like there must be something interesting behind it.

Thanks,

**Comment by aleneum** [comment]
Hello and happy new year @MichaelQuaMan,

interesting question!

> I’ve never seen this style. Looks like a conscious choice.

Phew, I never really thought about it. I see it occasionally in the wild but I guess it's not an actual style.

> I’d love to know why?

Personally, I just find it a bit easier to read without adding a newline. I cannot speak for Tal (initial author of transitions) though. He also used leading spaces after quotes.

> Seems like there must be something interesting behind it.

Maybe there is but then I don't know about it. As of now I'd say it's just me/us not following docstring conventions.
I should probably fix this the next time I review docstrings [#599].
